### PR TITLE
New id fixes

### PR DESCRIPTION
**Pull request by mjpost**
I found a few small bugs in testing out the latest ingest under the new ID format: https://www.aclweb.org/anthology/events/ws-2020/

- [x] PDFs copied to wrong place
- [x] Redirection for attachments has wrong rule
- [x] Nothing shows up on venue page (https://www.aclweb.org/anthology/venues/iwclul/)

I expect a few more to show up, so don't merge yet. I will look at this more tonight.

### CODE REVIEW

**Comment by mbollmann** [comment]
Have you added it to `venues.yaml`? That's still needed at this point since we haven't consolidated the files yet.

**Comment by mjpost** [comment]
Yes, it's there. The name from that file shows up [here](https://www.aclweb.org/anthology/venues/iwclul/), but it doesn't pull in the conference.

Oh, I bet this is because its primary venue is listed as "WS"? I should change it to "IWCLUL".

**Comment by mbollmann** [comment]
Right, if you want it to appear under both you need to use the `joint:` key, as I did in the test4nlp file.

**Comment by mjpost** [comment]
I’m just moving the listing to IWCLUL. I think we should move away from WS which is a meaningless catch all.

**Comment by mjpost** [comment]
Ah! But if it's not joint with WS, it won't get linked at all from the front page.

I guess I'll continue to use WS until we find a more general solution to that problem.

**Comment by mjpost** [comment]
This is also ready to be merged, if someone has a minute to review.

File: data/yaml/venues_mapping.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ---
 iwclul:
-  venue: WS
+  venue: IWCLUL
+  joint:
+    2020.iwclul: WS
```
Comment (mbollmann) [inline data/yaml/venues_mapping.yaml:5]: Have you tested this? I believe the key needs to point to a volume, i.e., `2020.iwclul-1`.

**Comment by mjpost** [comment]
Thank you!

**Pull request merged**